src/runtime/components/elements/Meter.vue

```diff
@@ -0,0 +1,187 @@
+<template>
+  <div :class="ui.wrapper" v-bind="attrs">
+    <template v-if="$props.indicator || $slots.indicator">
+      <slot name="indicator" v-bind="{ percent, value: $props.value }">
+        <div :class="indicatorContainerClass" :style="{ width: `${percent}%` }">
+          <div :class="indicatorClass">
+            {{ Math.round(percent) }}%
+          </div>
+        </div>
+      </slot>
+    </template>
+
+    <meter
+      :value="$props.value"
+      :min="normalizedMin"
+      :max="normalizedMax"
+      :class="[meterClass, meterAppearanceClass, meterBarClass]"
+    />
+
+    <template v-if="$props.label || $slots.label">
```

Comment:
Why use `$props` here?

Reply:
Just accustomed to use $props. Is that bad?

Reply:
You can just bind the `label` as long as it's returned in the setup. We don't use `$props` in any component of the lib.